src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('.tree');
+
+function f(listss) {
```

Comment:
Let's give function and its argument proper names. It's unclear what is f and why you have two 's' in listss.